career/expectations/software-development-analyst/README.md

```diff
@@ -0,0 +1,130 @@
+
+
+# Expectations
+
+What do we expect from a software development analyst at GeekHunter?
+
+
+## 🐣 Junior Software Development Analyst
+
+### Experience
+
+- **Usually goes with the current hype.** Always wants to try out new frameworks or languages but doesn't know how to balance this with the daily routine. Usually, this ends up hurting the productivity for himself or the team.
+- Knows at least a bit about software development (a few months to one year), preferably in production environments. Has already worked in new features, bugfixes, etc.
+- Knows at least a bit about agile culture, has a slight idea of what are key terms such as Scrum or Kanban
+- Does not review code for other collaborators. Usually finds it difficult to analyze external code and find problems in it.
+- Asks for help whenever they are feeling blocked, not productive or doesn't know what to do.
+- Usually codes "by example" (e.g. copying code), **does not understand how that code works**.
+- Requires more attention, since usually gets blocked often.
+- Is not proactive with new tasks. Usually needs someone else to tell them what to do.
+
+### Hard Skills
+
+- **Knows at least a bit of the tech stack that we use** (Ruby on Rails, React, Next.js, TypeScript, Python, GraphQL and others). Will eventually have blocks and doubts on how to develop with those technologies.
+- Knows at least a bit about automated tests, and at least understands their importance. Is capable of writing a test with the help of another developer.
+- Usually has a difficult time to produce code that respects our [Styleguide](https://github.com/GeekHunter-Brasil/hello-world).
+- Usually can't do TDD daily, but knows what it is.
+- Knows at least about git, but can't follow processes such as gitflow.
+- Doesn't know much about database technologies, and usually has a hard time to write a SQL query.
+- Knows about Object Oriented Programming, but has difficulty implementing it daily.
+- Usually **doesn't know much about software architecture**. Has a had time following our current pattern (e.g. clean or hexagonal architecture).
+- Doesn't know how to design an application according to its needs (a monolith, micro-services, serveless, etc). **Usually tries to implement a complex model where it's not needed and ends up hurting productivity**.
+- **Doesn't know how to debug code**. Usually has a difficult time trying to understand why something doesn't work.
+
+### Soft Skills
+
+- Is able to keep their presence in synchronous (e.g. squad rituals such as dailies, plannings, reviews; pair programmings) and asynchronous moments (e.g. discussions on Slack or Basecamp).
+- Is really motivated to learn more and develop their skills.
+- Usually doesn't understand their own strong or weak points, or how to proceed in their career. Needs an active leadership helping them to progress.
+
+## 🐤 Software Development Analyst
```

Comment:
I thought about this but decided to go with `Software Development Analyst` only, since the way we reference `Pleno` globally is just the position (think of it as a 100%).

Check https://outliers.com.br/2020/03/10/pleno-em-ingles/

Reply:
Based on the skills sets and expectations, I'd say our Engineering team is made of Software Engineers rather than Software Development Analysts. What you guys this?

https://www.computerscience.org/resources/software-developer-vs-software-engineer/

Reply:
Makes sense, @joaophellip!

Let's update this

Reply:
I agree with you @joaophellip, but I don't think we should make this change because of CLT standards.

Let's leave this as-is for now, we'll review later.